### PR TITLE
Plan migration of scripts to GitHub

### PR DESCRIPTION
**Pull request by fkautz**
Please review the file and leave feedback.

### CODE REVIEW

File: plan-migrate-scripts-to-github [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+3.  [x] Add `hachyderm-infra` to github org
+4.  [x] Create ssh keys and add them to the account
+
+## Create repository and copy scripts
+1.  [ ] Notify #mastodon-infra
+2.  [ ] Take a ZFS snapshot
+3.  [ ] Checkout repository to `/opt/scripts`
+4.  [ ] Set permissions to root:root 755
+5.  [ ] Copy `/home/novix/zfs` to `/opt/scripts/zfs`
+6.  [ ] Copy /var/lib/mastodon/run-* to `/opt/scripts/mastodon`
```
Comment (kris-nova) [inline plan-migrate-scripts-to-github:17]: Ideally we add these run scripts to $PATH and make them non-directory specific? 

eg: we have a `/bin` directory in `/opt/scripts/bin` and we just add that $PATH?

File: plan-migrate-scripts-to-github [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+7.  [ ] Submit each script family as a set of pull requests. e.g. zfs, mastodon
+
+## Create README describing how the process will work and describing what is linked
+1.  [ ] Fill the README.md and submit as a PR.
+
+## Once scripts are in the repository
+
+The intention is to swap each script with a softlink.
+
+e.g. `/var/lib/mastodon/run-script` would be softlinked to `/opt/scripts/mastodon`
```
Comment (kris-nova) [inline plan-migrate-scripts-to-github:27]: Soft linked?

Comment (fkautz) [reply plan-migrate-scripts-to-github:27]: You might know it as a symbolic link or symlink.

For anyone who comes across this:
* `ln -s target_path link_path` generates a symbolic link where the destination is listed in a regular file on the file system. Works across separate file systems.
*  `ln target_path link_path` generates a hard link, where the index in the filesystem node points directly to the data. Cannot traverse to separate mounted filesystems.

File: plan-migrate-scripts-to-github (hunk 1 above)
Comment (fkautz) [reply plan-migrate-scripts-to-github:17]: I'm happy to do this as well. Initial intention was to just get things moved over, then we can look at normalizing the scripts and removing it. I'll modify the plan to reflect this.

File: plan-migrate-scripts-to-github (hunk 2 above)
Comment (kris-nova) [reply plan-migrate-scripts-to-github:27]: :)

**Comment by kris-nova** [comment]
LGTM

**Pull request merged**